environment.yml

```diff
@@ -87,7 +87,7 @@ dependencies:
   - gitdb
   - natsort  # DataFrame.sort_values doctest
   - numpydoc
-  - pydata-sphinx-theme<0.11
+  - pydata-sphinx-theme>=0.13.0
```

Comment:
I think it's better to simply remove the version here, it should pick the last one anyway.